docs/process/metrics.md

```diff
@@ -72,3 +72,5 @@ These are general metrics about feature usage and specific feature behaviors. Th
 | `unattributedCommits` | The number of commits that will go unattributed to GitHub users. | To understand how frequently commits in GitHub Desktop are unattributed and how highly we should prioritize design for those instances |
 | `unguidedConflictedMergeCompletionCount` | The number of times a conflicted merge is completed from the diff/changes view. | To understand how many times people prefer to finish the merge in the changes/diff view after resolving conflicts. |
 | `updateFromDefaultBranchMenuCount` | The number of times the `Branch -> Update From Default Branch` menu item is used. | To understand usage patterns around the compare branches feature. |
+| `stashEntriesCreatedOutsideDesktop` | The number of stash entries created outside of Desktop | To gain insight into how much of an impact Desktop has reducing the number of stash entries created. |
+| `errorWhenSwitchingBranchesWithUncommmittedChanges` | The number of times the user is presented with the error message "Some of your changes would be overwritten" | To understand the impact stashing would have in reducing the number of errors on branch checkout. |
```

Comment:
```suggestion
| `errorWhenSwitchingBranchesWithUncommmittedChanges` | The number of times the user is presented with the error message "Some of your changes would be overwritten" | To understand the potential impact of stashing in reducing the number of errors experienced when switching branches. |
```

Reply:
Addressed in https://github.com/desktop/desktop/pull/7525/commits/a33fd3c2979c63f140a6aadbc394599512572dd0.